Composer/packages/lib/shared/src/types/server.ts

```diff
@@ -5,8 +5,7 @@ export interface ProjectTemplate {
   id: string;
   name: string;
   description: string;
-  /** Absolute path of the template */
-  path?: string;
-  /** Optional order property */
-  order?: number;
+  path: string; // absolute path
+  tags?: string[]; // tags for further grouping and search secenario
+  support?: string[]; // list of supported runtime versions
```

Comment:
TS nit: Move these comments above the property and wrap in `/** */` comment blocks so that TS tooling will pick up as descriptions.

Reply:
nice, thanks